data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -3,6 +3,19 @@ set -e
 
 mkdir --parents /etc/kubernetes/{manifests,bootstrap-configs,bootstrap-manifests}
 
+if ! podman inspect {{.ReleaseImage}} &>/dev/null; then
+    echo "Pulling release image"
+    podman pull {{.ReleaseImage}}
+fi
+
+if ! grep -q 'Installing release image' /etc/motd; then
```

Comment:
I'm not sure that this needs to be added to the MOTD. I think it would be more appropriate to just echo the version into the logs.

Reply:
Mmm...it feels important-ish (and the bootstrap node is just going to go away anyways).  

It looks like we already have it in the logs actually, so there's not much point to this PR if we don't do the motd.

Though while I'm here, one question; am I right in my reading of this code in that it relied (and still does) on the `MACHINE_CONFIG_OPERATOR_IMAGE=$(podman run --rm {{.ReleaseImage}} image machine-config-operator)` line pulling the image, and nothing else pulling subsequently?

Reply:
Yes, it relies on that first `podman run` to pull the image. Your PR makes it a bit more clear what is actually going on and I'd like to pull in that part.

Reply:
OK, I split that out to https://github.com/openshift/installer/pull/764